BrainPortal/app/controllers/bourreaux_controller.rb

```diff
@@ -88,7 +88,7 @@ def info #:nodoc:
   end
 
   def new #:nodoc:
-    bourreau_group_id = ( current_project && current_project.id ) || current_user.own_group.id
+    bourreau_group_id = default_target_group_id
```

Comment:
Why has the method's name changed to something that is even LESS descriptive than what it was before? `current_assignable_group_id` was perfectly fine.

Reply:
If it was better I revert. My line of thought was as follows: `default_project = current_project || current_user.own_group`  is already in codebase, and I thought it would be more descriptive, not in sense how method work but what goal it achieves. Also it is no always current, but sometimes just an assignable branch.  Having a different name might one day be a step to enhance logic behind defaulting target.